.travis.yml

```diff
@@ -157,7 +157,7 @@ script:
 - mkdir travis-build &&
   cd travis-build
 - cmake -DCMAKE_CXX_COMPILER="$COMPILER" $WARN $CMAKE_EXTRA_FLAGS ../
-- make -j
+- make
```

Comment:
This was in regards to Travis and the Nvidia compiler. It was eating up all the virtual memory and failing the test. @davidbeckingsale 's suggestion was to turn it off.